vllm/worker/worker_base.py

```diff
@@ -27,7 +28,7 @@
 logger = init_logger(__name__)
 
 
-class WorkerBase(ABC):
+class WorkerBase:
```

Comment:
you can just decorate this class

Reply:
The reason that I decorate `WorkerBase` in `vllm.v1.worker.worker_base` is that in addition to methods in `vllm.worker.worker_base.WorkerBase`, it will also remind your methods there (`vllm.v1.worker.worker_base.WorkerBase`)